cmd/brimcap/load/command.go

```diff
@@ -102,6 +119,31 @@ func (c *Command) Exec(args []string) (err error) {
 	return err
 }
 
+func (c *Command) writeBrimcapRoot(path string, f *os.File, warner zbuf.Warner) error {
+	index, err := pcap.CreateIndexWithWarnings(f, c.limit, warner)
+	if err != nil {
+		return err
+	}
+
+	b, err := json.Marshal(index)
+	if err != nil {
+		return err
+	}
+
+	indexPath := filepath.Join(c.rootflags.Root, filepath.Base(path)+".idx")
+	if err := os.WriteFile(indexPath, b, 0644); err != nil {
+		return err
+	}
```

Comment:
Maybe use github/brimdata/zed/pkg/fs.MarshalJSONFiile?

Reply:
I find myself wanting to limiting the amount of things that brimcap depends on from zed, but perhaps I am being unnecessarily cautious.

Reply:
But then again fs.MarshalJSONFile does also update atomically which would prevent (unlikely) read races. Sure I'll do it!